### PR TITLE
New UI BETA v1.1 Major Update

### PR DESCRIPTION
**Pull request by cannarelladev**
# Description

UI BETA v1.1: Major Update

## What's new

- [x] File Manager for Containers (Next Cloud link for VM)
- [x] Tooltip fix for Connect buttons
- [x] Modal alert for "Destroy All" function (mobile/desktop)
- [x] Info Sidebar typos fix
- [x] Grafana broken link fixed
- [x] Temporarily disabled "Manage User" Button
- [x] Added instance Prettyname in INFO button (Active Page [Manager View])
- [x] SSH refresh problems fix and some modal style improvements
- [x] PrettyName/Running commands no deterministic behavior fix
- [x] Various minor fixes and improvements
- [x] Copyable IP
- [x] User Filtering improvements (Now is possible to search both for User ID and User Prettyname
- [x] Edit "Prettyname" improvements

### CODE REVIEW

File: frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsExtended.tsx [hunk 1]
```diff
@@ -90,28 +94,27 @@ const RowInstanceActionsExtended: FC<IRowInstanceActionsExtendedProps> = ({
...
+          <span
+            className={`${fileManagerDisabled ? 'cursor-not-allowed' : ''}`}
+          >
+            <Button
+              shape="circle"
+              className={`hidden mr-3 xl:inline-block ${
+                fileManagerDisabled ? 'pointer-events-none' : ''
+              }`}
+              disabled={fileManagerDisabled}
+              onClick={() => window.open(`${url}/mydrive/files`, '_blank')}
```
Comment (QcFe) [inline frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsExtended.tsx:112]:
```suggestion
              href="`${url}/mydrive/files`"
              target="_blank"
```
It's better to use the link mode of the button. This also prevents issues with popup blockers and similar stuff.
I'd ask you to replace this behavior in other places if it's used anywhere else.

Comment (QcFe) [reply frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsExtended.tsx:112]: Also, do we want to try opening nextcloud with vms? (see [the slack](https://crown-team-group.slack.com/archives/C01J519B83G/p1636997434075600))

Comment (cannarelladev) [reply frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsExtended.tsx:112]: i fix it everywhere

Comment (cannarelladev) [reply frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsExtended.tsx:112]: ok for the suggestion about nextcloud

**Review by QcFe — APPROVED** [review]

**Pull request merged**